src/utils/index.ts

```diff
@@ -45,6 +45,22 @@ export function isActiveItem(router: Router, href: string, singlePage?: boolean)
     return normalizePath(router.pathname) === normalizePath(parse(href).pathname);
 }
 
+/*
```

Comment:
We recommend using English as the default language in open source projects.

Reply:
I have added the translation to it